Classes/PackageFactory/Guevara/Domain/Model/Changes/Create.php

```diff
@@ -1,9 +1,19 @@
 <?php
 namespace PackageFactory\Guevara\Domain\Model\Changes;
 
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Service\NodeTypeManager;
+
 class Create extends AbstractCreate
 {
     /**
+     * TODO: find out why everything breaks if I remove this dependency
```

Comment:
Hey people who are more into Flow, please help me understand why $this->nodeTypeManager in AbstractCreate.php becomes null (i.e. not injected), if I remove it from this file???